tests/lib/preseed.sh

```diff
@@ -47,6 +47,9 @@ setup_preseeding() {
     CORE_IMAGE=$(find "$IMAGE_MOUNTPOINT/var/lib/snapd/seed/snaps/" -name "core_*.snap")
     unsquashfs "$CORE_IMAGE"
     cp /usr/lib/snapd/snapd squashfs-root/usr/lib/snapd/snapd
+    # XXX to satisfy version check; this will go away once preseeding
+    # is available in 2.44
+    echo "VERSION=2.44.0" > squashfs-root/usr/lib/snapd/info
```

Comment:
(nitpick) I think this is no longer needed with the updated version check for 2.43.3+ - but not really worth another spread run, can be done in a followup.

Reply:
You're right. I'll leave for now but this entire function will go away very soon.